### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/6fa7a21c9f41f1a50d0cd1ca74daaede1d2b218f

### CODE REVIEW

File: en/firefox/new/desktop.ftl [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+firefox-desktop-download-download-the-mobile = Download the { -brand-name-firefox } mobile browser for automatic protection on all your devices.
+firefox-desktop-download-learn-about-the = Learn about the { -brand-name-firefox } desktop browser below
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+# "just right" could also be translated as "the way you want it to be" or "perfect for you"
+firefox-desktop-download-from-security-to = From security to news to gaming, there’s an <a { $attrs }>extension for everyone</a>. Add as many as you want until your browser is just right.
+
+# Variables:
+#   $attrs (attrs) - link to https://www.mozilla.org/en-US/about/
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:125]: Remove `en-US` in the URL

File: en/firefox/new/desktop.ftl [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+firefox-desktop-download-firefox-was-created = { -brand-name-firefox } was created by <a {$attrs }>{ -brand-name-mozilla }</a> as a faster, more private alternative to browsers like { -brand-name-ie }, and now { -brand-name-chrome }. Today, our mission-driven company and volunteer community continue to put your privacy above all else.
+
+# Variables:
+#   $attrs (attrs) - link to https://www.mozilla.org/en-US/firefox/privacy/
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:129]: Remove `en-US` in the URL

File: en/firefox/new/desktop.ftl [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+firefox-desktop-download-as-the-internet = As the internet grows and changes, { -brand-name-firefox } continues to focus on your right to privacy  — we call it the <a { $attrs }>Personal Data Promise</a>: Take less. Keep it safe. No secrets. Your data, your web activity, your life online is protected with { -brand-name-firefox }.
+
+# Variables:
+#   $attrs (attrs) - link to https://accounts.firefox.com/signin
+firefox-desktop-download-firefox-is-available = { -brand-name-firefox } is available on all your devices; take your tabs, history and bookmarks with you. All you need is a <a { $attrs }>{ -brand-name-firefox } account</a>.
+
+# Variables:
+#   $attrs (attrs) - link to https://support.mozilla.org/en-US/kb/firefox-screenshots
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:137]: Remove `en-US` in the URL

File: en/firefox/new/desktop.ftl [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+firefox-desktop-download-almost-there = Almost there!
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+firefox-desktop-download-now-else = Now finish <strong>installing</strong> { -brand-name-firefox }.
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+firefox-desktop-download-now-windows = Now <strong>run</strong> the installer that just downloaded.
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
+firefox-desktop-download-now-mac = Now <strong>open</strong> the file that just downloaded.
+# Variables:
+#   $attrs (attrs) - link to https://support.mozilla.org/en-US/products/firefox/download-and-install
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:164]: Remove `en-US` in the URL

File: en/firefox/new/desktop.ftl [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+firefox-desktop-download-clear-history = Clear history
+firefox-desktop-download-form-autofill = Form autofill
+firefox-desktop-download-automatic-updates = Automatic updates
+firefox-desktop-download-customize-your-browser = Customize your browser
+firefox-desktop-download-themes = Themes
+firefox-desktop-download-dark-mode = Dark mode
+firefox-desktop-download-library-of-extensions = Library of extensions
+firefox-desktop-download-adjust-search-bar = Adjust search bar settings
+firefox-desktop-download-change-new-tab = Change new tab layout
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
```
Comment (peiying2) [inline en/firefox/new/desktop.ftl:88]: No need for this comment for Firefox.

**Comment by craigcook** [comment]
Closed in favor of https://github.com/mozilla-l10n/www-l10n/pull/85

**Pull request closed without merging**